.travis.yml

```diff
@@ -14,6 +14,7 @@ before_install:
 - pip install coverage
 install:
 - pip install cli/[dev]
+- pytest --black
```

Comment:
can `black --check --diff` accomplish the same? what does `pytest` do that `black` doesn't?

Reply:
I used pytest because I didn't want Travis to run unless formatting is done properly. So what happens is if your code isn't formatted alright, Travis error will show for you to fix it before heading into the actual popper code.

Reply:
But now I typed that out loud, it may not be the best

Reply:
> I used pytest because I didn't want Travis to run unless formatting is done properly. So what happens is if your code isn't formatted alright, Travis error will show for you to fix it before heading into the actual popper code.

but `black --check --dif` does this, right? throw error if improperly formatted code is found?

Reply:
Yeah it does it and shows verbose. I didn't know of --diff gives verbose. Thanks for this